packages/shared/lib/services/connection.service.ts

```diff
@@ -1585,15 +1592,11 @@ class ConnectionService {
             NangoError
         >
     > {
-        // Note:
-        // a billable connection is a connection that is not deleted and has not been deleted during the month
-        // connections are pro-rated based on the number of seconds they were existing in the month
-
         const targetDate = new Date(Date.UTC(referenceDate.getUTCFullYear(), referenceDate.getUTCMonth(), referenceDate.getUTCDate(), 0, 0, 0, 0));
         const year = referenceDate.getUTCFullYear();
         const month = referenceDate.getUTCMonth() + 1; // js months are 0-based
 
-        const res = await db.readOnly
+        const q = db.readOnly
```

Comment:
we can await the query directly. I think it is a leftover of some early-on debugging